results_parser/ssl-cert.py

```diff
@@ -1,10 +1,16 @@
+import datetime
+
+
 def process_script(script, results_context):
     results = {}
     for table in script.table:
         if table["key"] == "validity":
             results["validity"] = {}
             for elem in table.elem:
                 results["validity"][elem["key"]] = elem.cdata
+                if elem["key"] == "notAfter":

```

Comment:
This method is a bit of a monster now. PLease consider splitting it up as a way to add more names and extend understanding